dotcom-rendering/src/components/ArticleMeta.stories.tsx

```diff
@@ -85,7 +74,7 @@ const defaultFormat: ArticleFormat = {
 	theme: Pillar.News,
 };
 
-export const ArticleStory: StoryObj = ({ format }: StoryArgs) => {
+export const Web: StoryObj = ({ format }: StoryArgs) => {
```

Comment:
Would having `Web` and `Apps` stories be a naming pattern we would like to introduce in general or is it just for the `ArticleMeta` stories? Just asking for people's opinion, not a question for this PR necessarily.

Reply:
In this case they are so widely different that I thought it was relevant. Hopefully most components are identical, otherwise there's little value in sharing the rendering platform 😜